element_calcium_imaging/imaging.py

```diff
@@ -1560,13 +1560,13 @@ class MaskMetrics(dj.Part):
         -> master
         -> Segmentation.Mask
         ---
-        mask_npix (int): Number of pixels in ROIs.
+        mask_size (float): Mask area in micrometer^2.
```

Comment:
syntax error

Reply:
I changed the variable name to mask_area and fixed the syntax error.